packages/shared-business/src/components/AccountStatementFooter.tsx

```diff
@@ -57,14 +52,6 @@ export const AccountStatementFooter = ({ language }: { language?: SupportedLangu
     <div style={styles.container}>
       <div style={styles.wrapper}>
         <div style={styles.footer}>{t("common.statement.footer")}</div>
-        <div style={styles.support}>
```

Comment:
I think we should also delete related keys in `styles` object :
- `support`
- `supportText`
- `supportLink`
- `pageNumber`